receiver/lokireceiver/factory.go

```diff
@@ -12,6 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+//go:generate mdatagen metadata.yaml
+
```

Comment:
Can you use the generated metadata.Stability here? I notice the level at line 34 is different than what the file specifies